aiida_vasp/workchains/vasp.py

```diff
@@ -89,6 +89,12 @@ def define(cls, spec):
                    help="""
             If True, enable more detailed output during workchain execution.
             """)
+        spec.input('dynamics.selective_dynamics',
```

Comment:
Could me maybe just do `dynamics.positions_dof` or something like that which helps us to know what it actually is (regardless of VASP)?

Reply:
Sure thing, will do that!